README.md

```diff
@@ -17,7 +17,7 @@ Company                                           | Business/products
 [10up](https://10up.com/)                         | Web consulting                           | WordPress, PHP, Node, Web                      | ✗                                    | Anywhere                | :keyboard: [Jobs](https://10up.com/careers/)<br>:door: [Glassdoor](https://www.glassdoor.com.au/Overview/Working-at-10Up-EI_IE775906.11,15.htm)
 [Airpush](https://www.airpush.com/)               | Mobile Advertising                       | Go, Java, PHP, Angular, Python, Web, Mobile    | ✓                                    | Anywhere                | :keyboard: [Jobs](https://airpush.com/careers/)<br>:door: [Glassdoor](https://www.glassdoor.com/Jobs/Airpush-Jobs-E789203.htm)
 [Auth0](https://auth0.com)                        | Identity Management                      | Node, Web                                      | ✓                                    | [7 countries](https://auth0.com/careers/positions?areas=Engineering)                       | :keyboard: [Jobs](https://auth0.com/jobs)<br>:door: [Glassdoor](https://www.glassdoor.com.au/Overview/Working-at-Auth0-EI_IE1162798.11,16.htm)
-[Automattic](https://automattic.com)              | WordPress & more                         | PHP, Java, Web, Mobile, Node                   | ✓                                    | Anywhere                | :keyboard: [Jobs](https://automattic.com/work-with-us/)<br>:door: [Glassdoor](https://www.glassdoor.com.au/Overview/Working-at-Automattic-EI_IE751107.11,21.htm)
+[Automattic](https://automattic.com)              | WordPress & more                         | PHP, Java, Web, Mobile, Node                   | ✓                                    | Anywhere                | :keyboard: [Jobs](https://automattic.com/work-with-us/)<br>:door: [Glassdoor](https://www.glassdoor.com.au/Overview/Working-at-Automattic-EI_IE751107.11,21.htm)<br>:open_hands: [Open Source](https://github.com/Automattic)
```

Comment:
I think it's better to keep open source links out of this list. It's already busy enough. :slightly_smiling_face: